src/dataframe/sort.jl

```diff
@@ -12,7 +12,9 @@ end
 function Base.sort!(df::DataFrame, a::Base.Sort.Algorithm, o::Base.Sort.Ordering)
     p = sortperm(df, a, o)
     pp = similar(p)
-    for col in columns(df)
+    c = columns(df)
+    uc = [!any(j->c[i] ≡ c[j], 1:i-1) for i=1:length(c)]
```

Comment:
It would be slightly more efficient to compute this directly in the loop, and `continue` for columns which have already been sorted.

Also, I don't think the `≡` is used anywhere in the code base, better keep the ASCII version for now. Finally, a comment would be useful.